diff -up vendor/dasprid/enum/src/AbstractEnum.php.patch vendor/dasprid/enum/src/AbstractEnum.php
--- vendor/dasprid/enum/src/AbstractEnum.php.patch	2025-09-12 11:28:28.712372237 +0200
+++ vendor/dasprid/enum/src/AbstractEnum.php	2025-09-12 12:02:48.595717515 +0200
@@ -214,7 +214,7 @@ abstract class AbstractEnum
      *
      * @throws SerializeNotSupportedException
      */
-    final public function __sleep() : array
+    final public function __serialize() : array
     {
         throw new SerializeNotSupportedException();
     }
@@ -224,7 +224,7 @@ abstract class AbstractEnum
      *
      * @throws UnserializeNotSupportedException
      */
-    final public function __wakeup() : void
+    final public function __unserialize(array $data) : void
     {
         throw new UnserializeNotSupportedException();
     }
