diff -up vendor/soundasleep/html2text/src/Html2Text.php.patch vendor/soundasleep/html2text/src/Html2Text.php
--- vendor/soundasleep/html2text/src/Html2Text.php.patch	2021-12-07 08:29:17.500834976 +0100
+++ vendor/soundasleep/html2text/src/Html2Text.php	2021-12-07 08:29:29.577097261 +0100
@@ -228,7 +228,7 @@ class Html2Text {
 		return $nextName;
 	}
 
-	static function iterateOverNode($node, $prevName = null, $in_pre = false, $is_office_document = false, $options) {
+	static function iterateOverNode($node, $prevName = null, $in_pre = false, $is_office_document = false, $options = []) {
 		if ($node instanceof \DOMText) {
 		  // Replace whitespace characters with a space (equivilant to \s)
 			if ($in_pre) {
