diff -up vendor/smarty/smarty/libs/sysplugins/smarty_internal_runtime_codeframe.php.patch vendor/smarty/smarty/libs/sysplugins/smarty_internal_runtime_codeframe.php
--- vendor/smarty/smarty/libs/sysplugins/smarty_internal_runtime_codeframe.php.patch	2021-10-11 16:02:02.854032181 +0200
+++ vendor/smarty/smarty/libs/sysplugins/smarty_internal_runtime_codeframe.php	2021-10-11 16:02:17.246356570 +0200
@@ -45,7 +45,7 @@ class Smarty_Internal_Runtime_CodeFrame
             $properties[ 'cache_lifetime' ] = $_template->cache_lifetime;
         }
         $output = "<?php\n";
-        $output .= "/* Smarty version {$properties[ 'version' ]}, created on " . strftime("%Y-%m-%d %H:%M:%S") .
+        $output .= "/* Smarty version {$properties[ 'version' ]}, created on " . date("Y-m-d H:i:s") .
                    "\n  from '" . str_replace('*/', '* /', $_template->source->filepath) . "' */\n\n";
         $output .= "/* @var Smarty_Internal_Template \$_smarty_tpl */\n";
         $dec = "\$_smarty_tpl->_decodeProperties(\$_smarty_tpl, " . var_export($properties, true) . ',' .
