diff -up vendor/symfony/string/AbstractString.php.patch vendor/symfony/string/AbstractString.php
--- vendor/symfony/string/AbstractString.php.patch	2025-09-12 11:14:56.496860038 +0200
+++ vendor/symfony/string/AbstractString.php	2025-09-13 06:42:36.260228693 +0200
@@ -706,9 +706,9 @@ abstract class AbstractString implements
         return $str;
     }
 
-    public function __sleep(): array
+    public function __serialize(): array
     {
-        return ['string'];
+        return ['string' => $this->string];
     }
 
     public function __clone()
diff -up vendor/symfony/string/UnicodeString.php.patch vendor/symfony/string/UnicodeString.php
--- vendor/symfony/string/UnicodeString.php.patch	2025-09-13 06:38:22.169394579 +0200
+++ vendor/symfony/string/UnicodeString.php	2025-09-13 06:43:56.651522624 +0200
@@ -362,8 +362,9 @@ class UnicodeString extends AbstractUnic
         return $prefix === grapheme_extract($this->string, \strlen($prefix), \GRAPHEME_EXTR_MAXBYTES);
     }
 
-    public function __wakeup(): void
+    public function __unserialize(array $data): void
     {
+        $this->string = $data['string'];
         if (!\is_string($this->string)) {
             throw new \BadMethodCallException('Cannot unserialize '.__CLASS__);
         }
