diff -up vendor/tecnickcom/tcpdf/include/tcpdf_images.php.patch vendor/tecnickcom/tcpdf/include/tcpdf_images.php
--- vendor/tecnickcom/tcpdf/include/tcpdf_images.php.patch	2022-05-03 20:56:17.330248199 +0200
+++ vendor/tecnickcom/tcpdf/include/tcpdf_images.php	2022-05-03 20:56:28.359497624 +0200
@@ -79,7 +79,7 @@ class TCPDF_IMAGES {
 			}
 		}
 		if (empty($type)) {
-            $type = strtolower(trim(pathinfo(parse_url($imgfile, PHP_URL_PATH), PATHINFO_EXTENSION)));
+            $type = strtolower(trim(pathinfo(parse_url($imgfile, PHP_URL_PATH) ?? '', PATHINFO_EXTENSION)));
 		}
 		if ($type == 'jpg') {
 			$type = 'jpeg';
