diff -up vendor/slim/twig-view/src/Twig.php.patch vendor/slim/twig-view/src/Twig.php
--- vendor/slim/twig-view/src/Twig.php.patch	2022-05-03 16:31:59.607830557 +0200
+++ vendor/slim/twig-view/src/Twig.php	2022-05-03 16:33:23.308745468 +0200
@@ -194,6 +194,7 @@ class Twig implements \ArrayAccess
      *
      * @return bool
      */
+    #[\ReturnTypeWillChange]
     public function offsetExists($key)
     {
         return array_key_exists($key, $this->defaultVariables);
@@ -206,6 +207,7 @@ class Twig implements \ArrayAccess
      *
      * @return mixed The key's value, or the default value
      */
+    #[\ReturnTypeWillChange]
     public function offsetGet($key)
     {
         return $this->defaultVariables[$key];
@@ -217,6 +219,7 @@ class Twig implements \ArrayAccess
      * @param string $key   The data key
      * @param mixed  $value The data value
      */
+    #[\ReturnTypeWillChange]
     public function offsetSet($key, $value)
     {
         $this->defaultVariables[$key] = $value;
@@ -227,6 +230,7 @@ class Twig implements \ArrayAccess
      *
      * @param string $key The data key
      */
+    #[\ReturnTypeWillChange]
     public function offsetUnset($key)
     {
         unset($this->defaultVariables[$key]);
